index.html

```diff
@@ -14,13 +14,14 @@
         <a href=""><img src="img/logo.svg" alt="logo"></a>  
       </div>
       <div class="toggle-button" type="toogle">
-        <span class="menu-icon"><img src="img/Menu.png" alt="menu"></span>
+        <i class="fa fa-bars menu-icon"></i>
       </div>
       <ul class="menu">
-        <li><a href="#" title="Go to title">About</a></li>
-        <li><a href="#" title="Go to services">Services</a></li>
-        <li><a href="#" title="Go to contact">Contact</a></li>
+        <li class="menu-item"><a href="#about-myself" title="Go to title">Portfolio</a></li>
+        <li class="menu-item"><a href="#work" title="Go to services">About</a></li>
+        <li class="menu-item"><a href="#form" title="Go to contact">Contact</a></li>
```

Comment:
- [ ] Kindly make sure the `About` link directs the user to `about-myself` and `Portfolio` directs to the `work` part of the page.

  ```suggestion
        <li class="menu-item"><a href="#work" title="Go to title">Portfolio</a></li>
        <li class="menu-item"><a href="#about-myself" title="Go to services">About</a></li>
        <li class="menu-item"><a href="#form" title="Go to contact">Contact</a></li>
```